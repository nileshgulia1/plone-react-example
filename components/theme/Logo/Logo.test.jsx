import React from 'react';
import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store';
import { Provider } from 'react-intl-redux';

import Logo from './Logo';

const mockStore = configureStore();

describe('Logo', () => {
  it('renders a logo component', () => {
    const store = mockStore({
      intl: {
        locale: 'en',
        messages: {},
      },
    });
    const component = renderer.create(
      <Provider store={store}>
        <Logo />
      </Provider>,
    );
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
