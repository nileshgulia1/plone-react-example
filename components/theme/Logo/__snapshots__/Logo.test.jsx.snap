// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders a logo component 1`] = `
<a
  onClick={[Function]}
  style={Object {}}
  title="Site"
>
  <img
    alt="Plone Site"
    className="ui image"
    height={32}
    src="Logo.svg"
    title="Plone Site"
  />
</a>
`;
